operations/research-and-development/engineering/team/README.md

```diff
@@ -61,3 +40,32 @@ Below is how SET prioritizes what is worked on.
    * Pick up any bugs in triage without an obvious owning feature team.
    * Work on bugs in this order: hotfix, release, customer reported, other.
 4. [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel — this channel should be monitored, but checking it once or twice per day is sufficient.
```

Comment:
Should we monitor other channels that community tends to use in the same way as this one? e.g.: "Contributors" and "Peer to Peer Help"